src/components/RadioGroup/RadioGroup.stories.mdx

```diff
@@ -0,0 +1,338 @@
+import RadioGroup from './RadioGroup';
+import { action } from '@storybook/addon-actions';
+import { withA11y } from '@storybook/addon-a11y';
+import { Meta, Story, Preview, Props } from '@storybook/addon-docs/blocks';
+import { useState } from 'react';
+
+<Meta
+  title="Components/Form Inputs/RadioGroup"
+  component={RadioGroup}
+  decorators={[withA11y]}
+/>
+
+# RadioGroup
+
+Use a RadioGroup when a user is required to select one of five or less options. It is ideal for this scenario because the options are displayed without having to interact. If there are more than five options, use the [SelectInput](?path=/docs/components-form-inputs-selectinput--all) instead.
+
+## Props
+
+<Props of={RadioGroup} />
+
+## Default
+
+All that is required to render a basic version of the RadioGroup is the group's `name`, an onchange event handler passed to the `onChange` prop, and a list of options passed to the `options` prop. Each option must include a unique `id`, a `value`, and a `name`.
+
+<Preview isExpanded>
+  <Story name="Default">
+    {() => {
+      const [value, setValue] = useState(null);
+      const options = [{
+        id: 'chocolate',
```

Comment:
The copy above says we need `id` `value` and `name` but these don't have `name` properties in the options

Reply:
Thanks for the careful proofing; appreciate it!